teams/lang-advisors.toml

```
@@ -0,0 +1,30 @@
+name = "lang-advisors"
+subteam-of = "lang"
+
+[people]
+members = [
+    "alexcrichton",
+    "Amanieu",
+    "cramertj",
+    "jackh726",
+    "JakobDegen",
+    "RalfJung",
+    "simulacrum",
+    "wesleywiser",
+]
+
+# Granting these permissions because an advisor can use these to help when
+# reviewing a proposed change.
+[permissions]
+perf = true
+crater = true
```

I think `@bors try` needs to be granted separately

```suggestion
crater = true
bors.rust.try = true
```